IFC4x3/Properties/c/ContactWireUplift_2SxilVeerCTud05s6E45Pj/DocProperty.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="ContactWireUplift_2SxilVeerCTud05s6E45Pj" Name="ContactWireUplift" UniqueId="9ceecbdf-a28d-4c77-89c0-17618e10566d" PrimaryDataType="IfcLengthMeasure" />
```

Comment:
If vertical upward movement, would you change ti `IfcPositiveLenghtMeasure`?

Reply:
Updated to IfcNonNegativeLengthMeasure: https://github.com/bSI-InfraRoom/IFC-Specification/pull/190/commits/be731fb1169a876200741df2c81db4c91e730337